### ISSUE TITLE
Quenstion about main_vct.py

### ISSUE THREAD

**Issue by wangjiakun13** [ThomasMrY/VCT#1]
hi, first congrats to your awesome work, I am confused about a piece of code,  what does this part do?  line 417-437 in main_vct.py
` if bs > 16:
                    with torch.no_grad():
                        image_folder = os.path.join(logger.get_dir(),"images")
                        os.makedirs(os.path.expanduser(image_folder), exist_ok=True)
                        sample_num = 16
                        latent1 = my_latents[:sample_num]
                        latent2 = my_latents[sample_num]
                        imgs_list = []
                        for i in range(latent_encoder.latents.shape[0]):
                            swapped_latent = latent1.clone()
                            swapped_latent[:,i] = latent2[None, i].repeat(sample_num,1)
                            opred = vct_dec(swapped_latent)
                            swap_imgs = decode_image(model, opred, old_shape)
                            imgs_list.append(swap_imgs)
                        imgs_record = torch.cat([data[:sample_num].detach().cpu(), data[sample_num][None,:,:,:].repeat(sample_num,1,1,1).detach().cpu()] + [rec.detach().cpu() for rec in imgs_list], dim=0)
                        imgs_record = torch.clip((imgs_record+1.)/2, 0.0, 1.0)
                        if imgs_record.shape[1] != 3:
                            imgs_record = imgs_record.repeat(1,3,1,1)
                        imgs_record = make_grid(imgs_record, nrow=sample_num).permute(1, 2, 0)
                        imgs_record = imgs_record.numpy()*255
                        imwrite("%s/s%08d_split.png" % (image_folder,it), imgs_record)`
